clients/v1_dbrps/v1_dbrps.go

```diff
@@ -0,0 +1,246 @@
+package v1dbrps
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.DBRPsApi
+	api.OrganizationsApi
+}
+
+type dbrpPrintOpts struct {
+	dbrp  *api.DBRP
+	dbrps []api.DBRP
+}
+
+type ListParams struct {
+	clients.OrgParams
+	ID       string
+	BucketID string
+	Default_ bool
```

Comment:
Why the trailing underscore here? And in the other structs

Reply:
Partially a hold-over from when I was using lower-cased property names (which didn't work, of course) and a conflict with the reserved `default` keyboard, and partially because the codegen apparently names the property `Default_`. I changed all the property names to just be `Default` and added a comment to where the property is assigned to the codegen'd request object.